src/main.js

```diff
@@ -1,32 +1,34 @@
-
-window.addEventListener('load', function() {
-  var firstCover = createRandomCover();
-  updateCover(firstCover);
-});
-
+// **** Global variables, general ****
 var savedCovers = [
   new Cover("http://3.bp.blogspot.com/-iE4p9grvfpQ/VSfZT0vH2UI/AAAAAAAANq8/wwQZssi-V5g/s1600/Do%2BNot%2BForsake%2BMe%2B-%2BImage.jpg", "Sunsets and Sorrows", "sunsets", "sorrows")
 ];
+var visibleCover = {};
```

Comment:
I'm not sure you need this. You could declare it inside of the function that reassigns it because it's the return value for that function. I could be wrong.

Reply:
yes, we still need to clean things up. will do in iteration 5.